client/src/lib/uiLib/components/DrawerMenu.tsx

```diff
@@ -1,43 +1,40 @@
 import React, {Fragment} from 'react';
 import {mapObjIndexed, mergeAll, values} from 'ramda';
-import {Text, ScrollView, Button, View, StyleSheet} from 'react-native';
+import {Text, ScrollView, Button, View} from 'react-native';
 import {NavigationContainer} from '@react-navigation/native';
 import {
   createDrawerNavigator,
   DrawerContentComponentProps,
 } from '@react-navigation/drawer';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import styled from 'styled-components';
 
 import {TouchableOpacity} from 'react-native-gesture-handler';
 import {useUiLib} from '../hooks/useUiLib';
 import {ComponentLibrary, ComponentList} from './UiLibRootComponent';
-import {SafeAreaView} from 'react-native-safe-area-context';
 import {COLORS} from '../../../common/constants/colors';
 
 const Drawer = createDrawerNavigator();
 
-const styles = StyleSheet.create({
-  wrapper: {flex: 1},
-  scrollWrapper: {flex: 1},
-  topLevelItem: {
-    fontSize: 17,
-    paddingLeft: 20,
-    fontWeight: 'bold',
-  },
-  menuItem: {
-    paddingVertical: 5,
-    backgroundColor: 'transparent',
-  },
-  menuItemActive: {
-    backgroundColor: COLORS.LEAF300,
-  },
-  menuItemText: {
-    fontSize: 15,
-    paddingLeft: 40,
-    color: 'black',
-  },
-  menuItemTextActive: {
-    color: 'white',
-  },
+const SafeAreaViewWrapper = styled(SafeAreaView)({flex: 1});
+
+const ScrollViewWrapper = styled(ScrollView)({flex: 1});
```

Comment:
You actually don't need to import components from `react-native`. Here you can instead write `styled.ScrollView({flex: 1});`